subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -876,7 +875,15 @@ static void isr_rx(void *param)
 
 	/* Set the channel to use */
 	if (!bis) {
-		data_chan_use = lll->ctrl_chan_use;
+		const uint16_t event_counter =
+				(lll->payload_count / lll->bn) - 1U;
+
+		/* Calculate the radio channel to use for ISO event */
+		data_chan_use = lll_chan_iso_event(event_counter, data_chan_id,
+						   lll->data_chan_map,
+						   lll->data_chan_count,
+						   &lll->data_chan_prn_s,
+						   &lll->data_chan_remap_idx);
```

Comment:
Same here.

Reply:
Same. However I will not be testing this on target.